vochain/state/state_test.go

```diff
@@ -175,6 +176,7 @@ type Listener struct {
 
 func (l *Listener) OnVote(vote *Vote, txIndex int32)                                             {}
 func (l *Listener) OnNewTx(tx *vochaintx.Tx, blockHeight uint32, txIndex int32)                  {}
+func (l *Listener) OnBeginBlock(req abcitypes.RequestBeginBlock)                                 {}
```

Comment:
Umm, I was trying to abstract the listeners from all the tendermint messy types. That's why all event handler functions contain only our types and usefull information.

So instead of passing the entire abcitypes.RequestBeginBlock it would be better to pass only the relevant parameters that we need.: req.Header.Height, req.Header.Time, req.Header.DataHash and detach the listeners from potential tendermint API/types future changes.

Reply:
ack, I wondered about this but wasn't sure.

Reply:
done